### ISSUE TITLE
[Extension]: a1111-sd-webui-tagcomplete compatibility

### ISSUE THREAD

**Issue by Davikar** [vladmandic/automatic#1590]
### Issue Description

Getting this error when starting up the webui:

```
16:13:08-931965 ERROR    executing callback:
                         C:\AI\automatic\extensions\a1111-sd-webui-tagcomplete\scripts\tag_autocomplete_helper.py
                         ui_settings_callback: AttributeError
╭─────────────────────────────────────── Traceback (most recent call last) ───────────────────────────────────────╮
│ C:\AI\automatic\modules\script_callbacks.py:182 in ui_settings_callback                                         │
│                                                                                                                 │
│   181 │   │   │   t0 = time.time()                                                                              │
│ ❱ 182 │   │   │   c.callback()                                                                                  │
│   183 │   │   │   timer(t0, c.script, 'ui_settings')                                                            │
│                                                                                                                 │
│ C:\AI\automatic\extensions\a1111-sd-webui-tagcomplete\scripts\tag_autocomplete_helper.py:304 in on_ui_settings  │
│                                                                                                                 │
│   303 │   │   "tac_activeIn.negativePrompts": shared.OptionInfo(True, "Active in negative prom                  │
│ ❱ 304 │   │   "tac_activeIn.thirdParty": shared.OptionInfo(True, "Active in third party textbo                  │
│   305 │   │   "tac_activeIn.modelList": shared.OptionInfo("", "Black/Whitelist models").info("                  │
╰─────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
AttributeError: 'OptionInfo' object has no attribute 'html'
```

The author states that:

> It seems SD.Next has not added the OptionInfo html method automatic1111 uses for some settings descriptions.

### Version Platform Description

Version: 3bcca6f9
Windows 11, Pytorch 2.01, xformers 0.0.20, GTX 1080 Ti, Vivaldi.

### URL link of the extension

https://github.com/DominikDoom/a1111-sd-webui-tagcomplete

### URL link of the issue reported in the extension repository

https://github.com/DominikDoom/a1111-sd-webui-tagcomplete/issues/197

### Acknowledgements

- [X] I have read the above and searched for existing issues

**Comment by DominikDoom** [comment]
Some additional context: After my fix, it isn't required anymore since I modified that option to also use info() instead of html(), but I imagine it is still good to have in case it is used in my or another extension in the future.

The use case in automatic itself is for the ability to add custom HTML elements separate from the info text, e.g. it is used in this setting for the list:
![image](https://github.com/vladmandic/automatic/assets/34448969/259c9a92-848a-4479-90bf-a4a8da0a9f68)

**Comment by vladmandic** [comment]
sdnext is on-par with a1111 v1.3.2, html method has been added very recently and majority of a1111 install base does not have that capability.
i'll add html method soon, for now thanks for the workaround.

**Comment by DominikDoom** [comment]
> sdnext is on-par with a1111 v1.3.2, html method has been added very recently and majority of a1111 install base does not have that capability.

Yeah true, it was only merged in last week with v1.4.0 which I didn't check beforehand, so me switching back to info would've happened either way. It's perfectly fine for my use case, I just chose html at first because I didn't need/want the parentheses info adds around the text for that specific setting.

**Comment by vladmandic** [comment]
this has been resolved long time ago.
